### ISSUE TITLE
llvm-3.9 travis build is failing

### ISSUE THREAD

**Issue by bscarlet** [bscarlet/llvm-general#185]
- The travis build is failing because travis can't install the llvm-3.9-dev package.
- Travis can't install the llvm-3.9-dev package because it's configured in .travis.yml to use the "llvm-toolchain-precise" distribution, which currently contains not 3.9 but 4.0 packages.
- The "llvm-toolchain-precise-3.9" distribution ought to contain llvm-3.9-dev package, but doesn't
- The "llvm-toolchain-precise-3.9" distribution is currently empty, because the [jenkins build](http://llvm-jenkins.debian.net/job/llvm-toolchain-precise-3.9-binaries/) which populates it is failing
- The jenkins build fails with `objcopy: unrecognized option '--extract-dwo'`
- objcopy does not recognize "--extract-dwo" because binutils on precise is old
- There is an [llvm bug](https://llvm.org/bugs/show_bug.cgi?id=28737) for the failure to happen earlier at configure time instead of build time, but as yet there seems to be none for the jenkins build failure itself.

**Comment by cocreature** [comment]
Even if `llvm-toolchain-precise-3.9` would contain the necessary packages, we can’t use it because the travis container builds can only install packages from whitelisted repositories and this repo is [not yet whitelisted](https://github.com/travis-ci/apt-source-whitelist/issues/300).

**Comment by bscarlet** [comment]
We could switch to building & caching LLVM ourselves, I suppose.

**Comment by cocreature** [comment]
Yeah we could do that, but I assume that when LLVM-3.9 is released they’ll publish binary packages so we can probably just download them, unpack & set the path appropriately and thereby avoid having to build it ourselves.

**Comment by cocreature** [comment]
But if that doesn’t happen in the next few days we should definitely build it ourselves.
